packages/components/src/tabs/stories/index.story.tsx

```diff
@@ -20,7 +20,7 @@ import Tooltip from '../../tooltip';
 import Icon from '../../icon';
 
 const meta: Meta< typeof Tabs > = {
-	title: 'Components/Containers/Tabs',
+	title: 'Components/Navigation/Tabs',
```

Comment:
I think we should revert this change. See @mirka's [comment](https://github.com/WordPress/gutenberg/pull/66072#pullrequestreview-2365228539) about it, as to why `Tabs` and `TabPanel` make most sense in "Containers". 

Let's confirm with @WordPress/gutenberg-design, and in case Figma's outdated, it can be good to update it.

Reply:
Thanks @tyxla, I would revert this change to use tabs and tabpanel in `Containers` category.